tsconfig.base.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
- [ ] 1. since github isn't helping here, can you confirm that this file is an exact copypaste from the old tsconfig.json?
- [ ] 2. can you remove all the keys that are always overridden anyway? like include, exclude, outdir?

Reply:
yes it exactly the same
but now i remove the include and outdir fields